templates/article/article.js

```diff
@@ -78,24 +78,38 @@ function createArticleCarousel() {
   return container;
 }
 
+function updateSocialLink(e) {
```

Comment:
this is cool. Do we need this at a common place so we use it everywhere?

Reply:
This is for sharing specifically. I'll refactor if we find another place it is needed.